src/main.rs

```diff
@@ -308,6 +316,98 @@ fn configure_sdk(args: &CliArgs) -> Result<()> {
     Ok(())
 }
 
+fn sign_fragmented(
+    manifest: &mut Manifest,
+    signer: &dyn Signer,
+    init_pattern: &PathBuf,
+    frag_pattern: &PathBuf,
+    output_path: &PathBuf,
+) -> Result<()> {
+    // search folders for init segments
+    let ip = init_pattern.to_str().ok_or(c2pa::Error::OtherError(
+        "could not parse source pattern".into(),
+    ))?;
+    let inits = glob::glob(ip)
+        .map_err(|_e| c2pa::Error::OtherError("could not process glob pattern".into()))?;
+    let mut count = 0;
+    for init in inits {
+        match init {
+            Ok(p) => {
+                let mut fragments = Vec::new();
+                let init_dir = p.parent().unwrap();
```

Comment:
get rid of unwraps Ok to map_err or just ? if it very unlikely to fail.

Reply:
can't really fail because of earlier checks but I will cleanup.